### ISSUE TITLE
Datatoolbar: wider chip labels

### ISSUE THREAD

**Issue by dlabrecq** [patternfly/patternfly-react#3563]
Migrating Cost Management to use the new data toolbar and noticed an issue with the chip label width. 

For the "Tag" category below, the chips represent tags (tag, key, value pairs), which can be lengthy. (Note that "Project" is a different category.) The current chip implementation truncates the chip label so that it makes it difficult to read.

![Screen Shot 2020-01-23 at 2 06 25 PM](https://user-images.githubusercontent.com/17481322/73015703-81f24000-3dea-11ea-8bdd-93001448807b.png)

When the "Tag Key" label is omitted, it's not clear what the chip represents below. I feel that the label is important here, so ideally we could use a little more width.

![Screen Shot 2020-01-23 at 2 23 59 PM](https://user-images.githubusercontent.com/17481322/73016525-03969d80-3dec-11ea-9653-d766078902c9.png)

**Comment by tlabaj** [comment]
@dlabrecq we have this issue open to add a tooltip that would show all the text for a chip group label when the text overflows.  I think that would solve your issue here. 
cc @mcarrano

**Comment by dlabrecq** [comment]
Yes, a tooltip would help. Still, there's not a lot of room.

I'll defer to @mcarrano. If there's nothing to adjust here, feel free to close.

**Comment by mcarrano** [comment]
@tlabaj @mcoker Is there any way to allow the space allowed for the category label to grow to accommodate the length of the label (within reason)? What is this currently constrained by?

**Comment by mcoker** [comment]
@mcarrano the label has a [max-width of 120px](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/ChipGroup/chip-group.scss#L20) in the CSS.

**Comment by mcarrano** [comment]
@mceledonia please take a look. Do you think we should increase the max-width?

**Comment by mceledonia** [comment]
Yeah, I would assume the max-width was initially implemented to prevent a single long chip from taking over the whole chip group area... But I definitely think we can be more flexible there. Extending the max width is one option, maybe we can go up to something like 160px and try that? The tooltip is a good idea too, it should probably also work on tap for mobile.

We could maybe try something like having any chip which extends beyond a threshold (160px) switching to something like 50% of it's total length, but I'm not sure if that would be worth the effort.
